### DISCUSSION TITLE
Question about adding new data to current model

### DISCUSSION THREAD

**Discussion by kronee0516** [ultralytics/yolov5#7588] [Q&A] [answered]
Hi, 

let's say i have a trained custom model for detecting pig
now i found some new pig's pics and labels as extra information for this model
do i need to train a new model out with old + new data?
or i can just train the new data with the old model only?

or any professional way to deal with this situation

Thank you

**Comment by glenn-jocher** [comment, marked as answer]
@kronee0516 always train on all data. See https://en.wikipedia.org/wiki/Catastrophic_interference
